src/Sylius/Component/Inventory/Model/InventoryUnitInterface.php

```diff
@@ -33,16 +33,6 @@
     public function getStockable();
 
     /**
-     * @param StockableInterface $stockable
-     */
-    public function setStockable(StockableInterface $stockable);
```

Comment:
Cause the Component never uses it most likely will never use, we don't care how stockable is defined for the InventoryUnit, we just need it to be there.